consul/consul.go

```diff
@@ -1,32 +1,25 @@
 package consul
 
 import (
+	"errors"
 	"fmt"
 	"net/url"
 	"strings"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/allegro/marathon-consul/apps"
+	"github.com/allegro/marathon-consul/service"
```

Comment:
I'd say right the opposite, you don't want our domain types with business logic depend on infrastructure, external classes like consul.